lib/python/rose/suite_engine_procs/cylc.py

```diff
@@ -91,7 +92,7 @@ class CylcProcessor(SuiteEngineProcessor):
     SUITE_DB = "cylc-suite.db"
     SUITE_DIR_REL_ROOT = "cylc-run"
     TASK_ID_DELIM = "."
-    TIMEOUT = 5  # seconds
+    TIMEOUT = 60  # seconds
```

Comment:
Yes. The main fix here, however, is to ensure that `rose suite-scan` does not hang. Previously, it could hang indefinitely. Now, the default is to kill the `cylc scan` and SSH commands after the timeout (which is also configurable via the site/user `rose.conf`).